test/python/circuit/library/test_nlocal.py

```diff
@@ -790,6 +790,12 @@ def test_efficient_su2(self):
         expected = n_local(4, ["ry", "rz"], "cx", "reverse_linear", reps=3)
         self.assertEqual(expected.assign_parameters(circuit.parameters), circuit)
 
+    def test_efficient_su2_numqubits_equal1(self):
+        """Test the efficient SU(2) circuit."""
+        circuit = efficient_su2(1)
+        expected = n_local(1, ["ry", "rz"], [], "reverse_linear", reps=3)
```

Comment:
Nice idea for the comparison 👍🏻 I think you can skip these arguments though:
```suggestion
        expected = n_local(1, ["ry", "rz"], [])
```

Reply:
yep sorry should have caught that myself